### ISSUE TITLE
Non-interactive create

### ISSUE THREAD

**Issue by jpslav** [openstax/manifestly#5]
Update the `create` command to have a non-interactive version.

```
manifestly create --force [--add=all] [--save_as=myfile.manifest]
```

Valid options to `--add` would be `all` (case insensitive) or e.g. `openstax/tutor-js,openstax/tutor-server`

If we do in fact reuse the `save_as` option (`download` currently uses it), we should probably change the interactive create menu option from `(w)rite manifest` to `(s)ave manifest` to be consistent.

**Comment by kevinburleigh75** [comment]
Instead of requiring the creation of a comma-separated list, I'd suggest either allowing all non-option arguments to be shunted to `--add`, or at least all non-option arguments following `--add` before the next option argument.

**Comment by jpslav** [comment]
Closed by #8
